extensions/banner/runtime/src/main/java/io/quarkus/banner/runtime/BannerConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.banner.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "banner", phase = ConfigPhase.BUILD_TIME)
+public class BannerConfig {
```

Comment:
This can be moved to the deployment module

Reply:
moved